utils/src/main/java/by/khadasevich/utils/StringUtils.java

```diff
@@ -0,0 +1,15 @@
+package by.khadasevich.utils;
+
+import static org.apache.commons.lang3.math.NumberUtils.*;
```

Comment:
please try to avoid merge imports, declare only methods you're needed in your class